README.md

```diff
@@ -1,2 +1,32 @@
 # Hello-Microverse1
 this repo contains Hello-Microverse1 project's files.
+## Built With
+- html
+- css
+## Live Demo (if available)
+Live Demo Link
+## Getting Started
+Simply you can open index.html file with any browser and, you will see Hello Microverse.
```

Comment:
[OPTIONAL] Kindly explain how to go about this setup well because this can be vague to someone who is new to it.